### DISCUSSION TITLE
Type filter button missing

### DISCUSSION THREAD

**Discussion by willykelly** [code4recovery/12-step-meeting-list#583] [Q&A] [answered]
How do I make my Type filter search button reappear and include type in meeting details?

http://melodytea.042c700.netsolhost.com/meetings/?tsml-day=4

**Comment by brianw-area24** [comment, marked as answer]
Do your meetings have types selected? The only way I was able to recreate this on my test site was to take away all the types on all the meetings on the site. As soon as I edited a meeting and made sure it had a type, the Type filter search button came back for me.

Brian W.
